runtime/autoload/health/provider.vim

```diff
@@ -89,8 +90,15 @@ function! s:system(cmd, ...) abort
     call health#report_error(printf('Command timed out: %s', s:shellify(a:cmd)))
     call jobstop(jobid)
   elseif s:shell_error != 0 && !ignore_error
-    call health#report_error(printf("Command error (job=%d, exit code %d): `%s` (in %s)\nOutput: %s\nStderr: %s",
-          \ jobid, s:shell_error, s:shellify(a:cmd), string(getcwd()), opts.output, opts.stderr))
+    let error = printf("Command error (job=%d, exit code %d): `%s` (in %s)",
+          \ jobid, s:shell_error, s:shellify(a:cmd), string(getcwd()))
+    if !empty(opts.output)
+      let error .= "\nOutput: " . opts.output
+    end
+    if !empty(opts.stderr)
+      let error .= "\nStderr: " . opts.stderr
+    end
+    call health#report_error(error)
```

Comment:
generally the word "error" should be avoided, I would call it `err` or `e`.

Reply:
Thanks for telling me again.. :)
Used "emsg".